### ISSUE TITLE
Linux no sound for actions such as slide and round insertion.

### ISSUE THREAD

**Issue by Sweeper5** [CiarenceW/M3SUPER90_Plugin#3]
Hello, a really nice plugin you have, I will start off with saying I am not familiar with Receiver 2 modding.
However after getting the gun to spawn for me in the Mall / Firing range, I have an issue with this weapon.
Unlike the MP5N from you, the M3Super90 is silent when using the pump action or loading it, however the firing sound is present so it is not a super silent shotgun.

I tried looking through your source code for any hints as to how you reference the sounds as my first primary suspect is capitalization issues.
In case you or others that read this doesn't know:

m3_add_round.wav
M3_add_round.wav

Would be considered two completely different files, while under Windows they would be 1 and the same.

Thank you for the good work, and hopefully it is a quick fix and easy to narrow down.

**Comment by CiarenceW** [comment]
yeah I think what's happening is that the old version of this plugin that has Linux support uses an audio system that isn't supported anymore because of thunderstore & stuff. Use proton? fixed? idk
